src/screens/FeedScreen/ContentPoll.js

```diff
@@ -111,7 +111,7 @@ const ContentPoll = ({
             height: 4,
             borderRadius: 4,
             alignSelf: 'center',
-            backgroundColor: COLORS.blackgrey
+            backgroundColor: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.blackgrey` to `COLORS.gray400` for the background color seems fine as long as `COLORS.gray400` is defined and it meets the design requirements.
```diff
- backgroundColor: COLORS.blackgrey
+ backgroundColor: COLORS.gray400
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->